setup.py

```diff
@@ -112,17 +109,13 @@ def append_env(L, e):
     ext_modules=[pylibmc_ext],
     package_dir={'': 'src'},
     packages=['pylibmc'],
+    python_requires='>=3.5',
     classifiers=[
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.2',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
```

Comment:
...but no 3.8 in here.

Reply:
I had left it out as it's also in PR https://github.com/lericson/pylibmc/pull/257, but I think it makes sense to put it here too. Updated!